djangoessentials/djangoessentials.py

```diff
@@ -15,3 +17,38 @@ class MyS3Storage(S3Boto3Storage):
     location = 'media/'  # S3'te files alt dir
     file_overwrite = False  # if have a same name about the file cant overwrite
     default_acl = 'public-read' # just can read permission
+
+
+
+#CustomUser Model
+class CustomUserManager(BaseUserManager, TimeBasedStampModel):
```

Comment:
in here we dont need it to have `TimeBasedStampModel` bro <3

Reply:
Thank you. I thought it would be helpful for admin too.